content/millennium/r4/clinical/diagnostics/questionnaire-response.md

```diff
@@ -41,6 +41,7 @@ The following fields are returned if valued:
 
```

Comment:
once we return this extension QR read and search will no longer be beta, so we can remove the tag from those sections https://github.com/cerner/fhir.cerner.com/blame/fd1e5bc0fa24961f4b2451bf238b345d5724f80d/content/millennium/r4/clinical/diagnostics/questionnaire-response.md#L58

Reply:
https://github.com/cerner/fhir.cerner.com/pull/697/commits/ca0df6a51fe1bc45033cea19dcc23f30e78aa3b6#diff-65d395ab3c58d2ca0d3ac40728ec9e34347612b86c616594927d663c0ccb1c31L58